contrib/ruby/test/client_test.rb

```diff
@@ -369,7 +369,7 @@ def test_trilogy_closed?
   def test_read_timeout
     client = new_tcp_client(read_timeout: 0.1)
 
-    assert_raises Errno::ETIMEDOUT do
+    err = assert_raises Trilogy::TimeoutError do
```

Comment:
Nit: Doesn't look like `err` is used just yet, perhaps we assert on the message or remove the variable assignment